components/longSparkArrow/LongSparkArrow.styles.js

```diff
@@ -0,0 +1,107 @@
+import { P } from '../../style/typography';
+import styled from 'styled-components';
+import { colors } from '../../style/colors';
+import { fontWeights } from '../../style/typography';
+import { min } from '../../lib/responsive';
+
+const UnfilledSparkArrow = styled.img`
+  position: absolute;
+  transform: rotate(90deg);
+  z-index: 1;
+  @media ${min.mobile} {
+    top: -1.3em;
+    right: 12em;
+    height: 30em;
+  }
+  @media ${min.tablet} {
+    top: -1em;
+    right: 8.2em;
+    height: 21em;
+  }
+  @media ${min.desktop} {
+    top: 2.5em;
+    right: 5.7em;
+    height: 14em;
+  }
+`;
+
+const FilledSparkArrow = styled.img`
+  position: absolute;
+  transform: rotate(90deg);
+  z-index: 1;
+  @media ${min.desktop} {
+    top: 2.1em;
+    right: 8em;
+    height: 19em;
+  }
+`;
+
+const ArrowTextHover = styled(P)`
+  position: absolute;
+  z-index: 2;
+  right: 0;
+  text-align: right;
+  color: ${colors.BROWN};
+  font-weight: ${fontWeights.bold};
+  @media ${min.desktop} {
+    padding-top: 9.8em;
+    width: 20em;
+  }
+`;
+
+const ArrowText = styled(P)`
+  position: absolute;
+  z-index: 2;
+  right: 0;
+  text-align: right;
+  color: ${colors.BROWN};
```

Comment:
same suggestion as [here](https://github.com/Scout-NU/thealleyway-site/pull/84/files#r612103518) for less repetitive code